pkg/endpoint/restore.go

```diff
@@ -166,7 +166,8 @@ func (e *Endpoint) RegenerateAfterRestore() error {
 	scopedLog := log.WithField(logfields.EndpointID, e.ID)
 
 	regenerationMetadata := &regeneration.ExternalRegenerationMetadata{
-		Reason: "syncing state to host",
+		Reason:            "syncing state to host",
+		RegenerationLevel: regeneration.RegenerateWithoutDatapath, // Needs datapath load??
```

Comment:
During restore we don't know what the state of the datapath is so we at least need to rewrite.

```suggestion
		RegenerationLevel: regeneration.RegenerateWithDatapathRewrite,
```

Reply:
Changed as indicated.